com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfJoints/UrdfJoint.cs

```diff
@@ -45,7 +45,7 @@ public enum JointTypes
         public string jointName;
 
         public abstract JointTypes JointType { get; } // Clear out syntax
-        public bool IsRevoluteOrContinuous => JointType == JointTypes.Revolute || JointType == JointTypes.Revolute;
+        public bool IsRevoluteOrContinuous => JointType == JointTypes.Revolute || JointType == JointTypes.Continuous;
```

Comment:
I don't exactly know why this API is provided, but my guess is that this boolean is here because we are trying to provide a general API for both articulation body and rigid body, where the `Continuous` and `Revolute` both refer to the revolute joint